src/rtp/99_locals.tf

```diff
@@ -4,4 +4,6 @@ locals {
 
   apim_name = "${local.product}-apim"
   apim_rg   = "${local.product}-api-rg"
+
+  rtp_base_url = "https://api-rtp.${var.env}.${var.prefix}.pagopa.it"
```

Comment:
in case of prod environment the pattern should be different. Maybe recycle something like `dns_zone_prefix = "dev.cstar"` and place it in `terraform.tfvars` in each environment

Reply:
Fixed by https://github.com/pagopa/rtp-apis/pull/3/commits/c18260df3c182ba0252d8629c6412341dbe0b337